runtime/dart_vm_lifecycle.cc

```diff
@@ -41,7 +41,7 @@ DartVMRef::~DartVMRef() {
   vm_.reset();
 }
 
-DartVMRef DartVMRef::Create(Settings settings,
+DartVMRef DartVMRef::Create(const Settings& settings,
```

Comment:
This seems like a drive by change unrelated to the patch.

Reply:
Removed them. Done.